src/holoscan_cli/templates/module/{{cookiecutter.module_repo_name}}/README.md

```
@@ -101,20 +102,18 @@ cmake --build build -j$(nproc)
 ```bash
-./holohub test
+uv run holoscan test
 ```
 
-Or, without the HoloHub CLI:
-
-{% if cookiecutter.language == 'cpp' %}
+Or, without the Holoscan CLI:
 
+{% if cookiecutter.language == 'cpp' -%}
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)